confluent/docker_utils/cub.py

```diff
@@ -110,6 +110,7 @@ def check_zookeeper_ready(connect_string, timeout):
     cmd_template = """
              java {jvm_opts} \
                  -cp {classpath} \
+                 -Dlog4j.configuration=file:/etc/kafka/log4j.properties \
```

Comment:
We need to figure out the log4j property path for all images that use cub command.